mrp_sale_info/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Antiun Ingenieria S.L. - Javier Iniesta
```

Comment:
Should be `__manifest__.py` isn't it?

Reply:
Yes, something happened with the renaming commit...

Reply:
Changed